python/setup.py

```diff
@@ -1,11 +1,155 @@
 # Copyright (c) 2021-2022, NVIDIA CORPORATION. All rights reserved.
 # See file LICENSE for terms.
 
-from setuptools import find_packages
-from skbuild import setup
```

Comment:
Would try to prefer the changes in `branch-22.04` when resolving the conflicts here. In particular we need Vyas' changes ( https://github.com/rapidsai/kvikio/pull/32 ) like the lines here

Reply:
Honestly I would suggest checking out the version of `setup.py` from `branch-22.04` and building on that directly. AFAICT the only changes that should be made in this PR are to add nvcomp as a dependency for the Python library and to create the CMake files for the new nvcomp subpackage. @thomcom let me know if you'd like me to try to do that on this branch.

Reply:
Grr I intended to take only the branch-22.04 version of the file, not sure what happened. Retry coming up.